metomi/rose/apps/rose_ana_v1.py

```diff
@@ -168,23 +167,23 @@ def run(self, app_runner, conf_tree, opts, args, uuid, work_files):
                          popen=app_runner.popen)
 
         # Initialise a database session
-        rose_ana_task_name = os.getenv("ROSE_TASK_NAME")
+        metomi.rose_ana_task_name = os.getenv("ROSE_TASK_NAME")
```

Comment:
Does it actually work? I never tried setting a value in a module that is also using namespaces... sounds dangerous, but if that works :man_shrugging: :grin:

Reply:
Nope - that's the result of over-enthusiastic `sed` use I'm afraid. #sedthusiasm

Reply:
Well spotted!

Reply:
Haha, I think there is another place where this happened too, but I didn't comment. Let me add a comment there too.